File: uri_1168_led.py
cases = int(input())
for i in range(cases):
    value = input()
    leds = 0
    for c in range(len(value)):
        if value[c] == '1':
            leds += 2
        elif value[c] == '2':
            leds += 5
        elif value[c] == '3':
            leds += 5
        elif value[c] == '4':
            leds += 4
        elif value[c] == '5':
            leds += 5
        elif value[c] == '6':
            leds += 6
        elif value[c] == '7':
            leds += 3
        elif value[c] == '8':
            leds += 7
        elif value[c] == '9':
            leds += 6
        elif value[c] == '0':
            leds += 6
    
    print("{} leds".format(leds))

File: uri_1168_led.c
#include <stdio.h>
#include <string.h>

int main(void){
	int cases; int leds = 0; char value[10000];
	scanf("%d\n", &cases);
	int c = 0;
	for (c = 0; c < cases; c++){
		scanf("%s", value);
		int i = 0;
		for (i = 0; i < strlen(value); i++){
			switch(value[i]){
				case '1':
					leds += 2;
					break;
				case '2':
					leds += 5;
					break;
				case '3':
					leds += 5;
					break;
				case '4':
					leds += 4;
					break;
				case '5':
					leds += 5;
					break;
				case '6':
					leds += 6;
					break;
				case '7':
					leds += 3;
					break;
				case '8':
					leds += 7;
					break;
				case '9':
					leds += 6;
					break;
				case '0':
					leds += 6;
					break;			
			}
		}
		printf("%d leds\n", leds);
		leds = 0;
	}
	return 0;
}